.gitignore

```diff
@@ -78,6 +78,8 @@ seedutil/seedutil
 test/simulation/temp
 test/simulation/docker-xud/xud
 test/simulation/docker-xud/custom-xud
+test/simulation/go
+temp_file
```

Comment:
temp_file should be removed. Also, I don't have a `test/simulation/go` folder in my repo, I think you should be able to simply delete it and remove it from here as well.